### PR TITLE
[Coeurs Vaillants 2] Sheet update for game launch

### PR DESCRIPTION
**Pull request by ChocoLanoar**
# Submission Checklist
## Required

- [x] The pull request title clearly contains the name of the sheet I am editing.
- [x] The pull request title clearly states the type of change I am submitting (New Sheet/New Feature/Bugfix/etc.).
- [x] The pull request makes changes to files in only one sub-folder.
- [x] The pull request does not contain changes to any json files in the translations folder (translation.json is permitted)

# Changes / Description
## General
* correct indentation (html code)
* change sheet checkbox with drop-down

## PC sheet
* Rename peuple into origine.
* Rename capacité into métiers.
* Move don de naissance and origine down with other capacities.
* Set compensation below stats.
* Rename richesse into points de trésors.
* Set properly hp_max and aventure_max.

## NPC sheet
* Set properly the scale to take the full width.
* Have only one big textarea for capacités to allow copy-paste from bestiary.
* Add one more field for another attack.

### CODE REVIEW

File: CoeursVaillants-2/CoeursVaillants-2.html [hunk 1]
```diff
@@ -15,136 +22,140 @@
...
-                        <div class="sheet-mycol">
-                            <div class="sheet-infoDesc" style="display: block;">Temp.</div>
-                            <input type="number" name="attr_pvTemp" class='sheet-inpnum2c' style="width:40px;" value='0'/>
-                        </div>
-                        <div class="sheet-mycol">
-                            <div class="sheet-infoDesc" style="display: block;">Aggr.</div>
-                            <input type="number" name="attr_pvAggr" class='sheet-inpnum2c' style="width:40px;" value='0'/>
-                        </div>
+            <div class="sheet-libel1" style="width:10px; vertical-align: bottom;">/</div>
+            <input type="number" name="attr_pv_max" class='sheet-inpnum2c' style="width: 45px; margin-right: 20px; vertical-align: bottom" value='0' />
```
Comment (NorWhal) [inline CoeursVaillants-2/CoeursVaillants-2.html:117]: changing attributes like this should come with [version handling](https://wiki.roll20.net/Character_Sheet_Development/Pattern_Libraries#Sheet_Versioning) to convert the data on existing sheets and prevent destruction of data for existing games

File: CoeursVaillants-2/CoeursVaillants-2.html [hunk 2]
```diff
@@ -15,136 +22,140 @@
...
 				<div class="sheet-mycolbrd" style="width: 49%;">
 					<div class="sheet-ligne" style="margin-bottom: 0px; text-align: center;">
 						<div class="sheet-libel3" style="width: 200px;">Points d'Aventure</div>
 					</div>
 					<div class="sheet-ligne" style="margin-bottom: 0px; text-align: center;">
 						<input type="number" name="attr_aventure" class='sheet-inpnum2cbig' style="width: 40px; margin-right: 0px;" value='1' min="0">
-                        <div class="sheet-libel1" style="width: 5px; margin: 0px;">/</div>
-                        <input type="number" name="attr_aventureMax" class='sheet-inpnum2c' style="width: 35px; margin-right: 0px;" value='1' min="0"/>
+            <div class="sheet-libel1" style="width: 5px; margin: 0px;">/</div>
+            <input type="number" name="attr_aventure_max" class='sheet-inpnum2c' style="width: 35px; margin-right: 0px;" value='1' min="0"/>
```
Comment (NorWhal) [inline CoeursVaillants-2/CoeursVaillants-2.html:145]: same as above

**Review by NorWhal — CHANGES_REQUESTED** [review]
Bonjour, @ChocoLanoar!

I will need to block this until version handling is added in your [sheetworkers](https://help.roll20.net/hc/en-us/articles/360037773513-Sheet-Worker-Scripts#h_01HQ47SB2H5789WR6B96VXHVRJ) or the attributes listed below are changed back to their original names. This is to prevent the loss of data in existing games from a sheet update. I'll keep an eye out for an update, so tag me here once this is applied and I'll work to get it approved as soon as possible.

Nora

File: CoeursVaillants-2/CoeursVaillants-2.html (hunk 1 above)
Comment (ChocoLanoar) [reply CoeursVaillants-2/CoeursVaillants-2.html:117]: Sure, I didn’t anticipate it. I’m looking into sheetworker stuff to add a migration script.

**Comment by ChocoLanoar** [comment]
Hi @NorWhal , I added a sheetworker update management script. You can find it at the beginning of Coeurs-Vaillants-2.html in commit e3dd5b880679fc866e49442a6bcd3e3e5c04ac8c.

**Pull request merged**